### ISSUE TITLE
Run tests with Github Action

### ISSUE THREAD

**Issue by kshitijrajsharma** [hotosm/fAIr#36]
Plan is to run backend and frontend tests with github actions

**Comment by natrimmer** [comment]
@kshitijrajsharma the backend tests are running in a github action already, and I don't see any frontend testing. Is there more to this issue?

**Comment by omranlm** [comment]
I did see some Frontend build tests using different Ubuntu versions after my last PR 
https://github.com/hotosm/fAIr/actions/runs/11347609698

But Frontend Build are not showing on main branch here 
![image](https://github.com/user-attachments/assets/43c250a9-cbb6-47d3-b1dd-b5c137f35b13)

It might be related to making `develop` as main branch @kshitijrajsharma ?

**Comment by kshitijrajsharma** [comment]
Yes , Frontend test cases should be established and get it running from github actions , You are correct backend is done , Frontend is remaining ! Current github actions only check build for the frontend !

**Comment by natrimmer** [comment]
@kshitijrajsharma @omranlm 
Ok that makes sense! Thank you for the clarifications!

**Comment by kshitijrajsharma** [comment]
Opening this up , We can implement frontend test cases now !
